calendars/numpy.yaml

```diff
@@ -19,8 +19,8 @@ events:
       The fortnightly NumPy development triage call.
 
       Meeting notes: https://hackmd.io/68i_JvOYQfy9ERiHgXMPvg
-    begin: 2022-01-26 17:00:00 +00:00
-    end: 2022-01-26 18:00:00 +00:00
+    begin: 2022-03-23 16:00:00 +00:00
+    end: 2022-03-23 17:00:00 +00:00
```

Comment:
Will this remove the past meetings from the calendar? Not sure this is a problem but I was just wondering if we want to keep past meetings in the calendar or not... Maybe this is irrelevant as long as we have meeting notes.

Reply:
Sounds good then!